service_capacity_modeling/interface.py

```diff
@@ -683,7 +683,12 @@ class CurrentClusterCapacity(ExcludeUnsetModel):
     cluster_instance_name: str
     cluster_instance: Optional[Instance] = None
```

Comment:
I think we should either get rid of this field and look it up from our shapes + cluster_instance_name or dont have it as optional.

Reply:
Yeah I think we should just do name, agreed

Reply:
Actually I take this back. generate_scenarios fills this in rather than the caller (probably why its optional). I do see that this would be useful since the model doesnt have access to the hardware shapes. I did however move my helper call until after generate_scenarios fills in the field.